src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -748,12 +805,6 @@ class device {
         sensor_tree::put("debug_profile.device_info.nifd_name", std::string(info.nifd_name));
         /** End of debug and profile device information */
 
-        // p2p enable
```

Comment:
why removed this?

Reply:
Moved it up so that it is with the rest of board.info